docs/concepts/rate-limit-nft-concept.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 4
+---
+
+# Rate Limiting
+
+# Overview
+
+By default, all users get three free requests on Lit every 24 hours. In order to use the network beyond the rate limit, you must reserve additional capacity. This can be done using capacity credits, which allow holders to reserve a configurable number of requests (measured in requests per second) over a fixed length of time (i.e. one week).
+
+:::note
+Currently Rate Limiting is only enabled on `Habanero` and `Manzano`
+see [here](https://developer.litprotocol.com/v3/network/networks/testnet) for test networks
```

Comment:
for links, do we want to redirect or use local references? (i.e. /v3/network... vs https://developer...)